CHANGELOG.md

```diff
@@ -74,6 +74,10 @@ and this project adheres to
   mechanism to reliably fetch Firecracker PID. With this change, Firecracker
   process's PID will always be available in the Jailer's root directory
   regardless of whether new_pid_ns was set.
+- [#4468](https://github.com/firecracker-microvm/firecracker/pull/4468): Made
```

Comment:
Nit: maybe refer to the micro-http commit/PR in the commit message.

Reply:
added the PR link in the commit message! thanks!